### PR TITLE
RK-11190 - Improve Breakpoints Docs

### PR DESCRIPTION
**Pull request by lirook**
* Changed breakpoints\-tasks to breakpoint\-limits
* moved data\-redaction to a separate page
* improved and refreshed data\-redaction
* removed obselete bp\-structure and bp\-scripting page files
* Improved and refreshed the conditional breakpoints page
* Removed json stuff from the limit page - This still needs a lot of work
* Re-added and refreshed and improved the built-in variables in a separate page.

### CODE REVIEW

**Comment by lirook** [comment]
@eladuzan94 @jhendrick @ElinorSwery FYI

File: docs/breakpoint-limits.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (lirook) [inline docs/breakpoint-limits.md:1]: On this page, I only removed a couple of things such as the JSON reference, I didn't yet go over it and fix stuff.
This page needs a lot more work...

File: docs/breakpoint-limits.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Auto-disable after global hit count limit
+
+You can set a breakpoint to automatically disable after a global hit count is hit.
+
+This is a soft limit - once it is hit, the breakpoint will be disabled but it may take a short while for it to be received and handled by your app instances, so you may see more hits than your limit. 
+
+## Maximum running time
+
+As a matter of safety to your application, when a single breakpoint run takes more than a reasonable time to run, it is disabled permanently. When this happens, the breakpoint will need to be manually re-enabled, perhaps also collecting less data to prevent the safety from re-triggering.
```
Comment (ElDuderinos) [inline docs/breakpoint-limits.md:50]: This is the same as "Maximum execution time limit", you can remove one of them

File: docs/breakpoint-limits.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+If a Breakpoint reaches the time limit for a specific app instance, the breakpoint will be disabled for the instance, and the Breakpoint status will change to "Error".
+
+The default value for this configuration is 400ms.
+
+## Auto-disable after global hit count limit
+
+You can set a breakpoint to automatically disable after a global hit count is hit.
+
+This is a soft limit - once it is hit, the breakpoint will be disabled but it may take a short while for it to be received and handled by your app instances, so you may see more hits than your limit.
```
Comment (ElDuderinos) [inline docs/breakpoint-limits.md:46]: Why not add also the time limit?

File: docs/breakpoints.md [hunk 4]
```diff
@@ -1,21 +1,32 @@
...
-You can read more about the [Breakpoint structure](breakpoints-structure.md) and how to perform [common changes](breakpoints-tasks.md) to it.
+The next time the code you have set the breakpoint on is invoked, Rookout collects parts of the application state and sends it to the IDE or any other targets of your choosing.
+
+## Breakpoint Limits:
+
+You can set limits on individual breakpoints to limit the amount of data to be collected. When limits are reached, the breakpoint should be disabled automatically. Once disabled, it should not collect additional data. You can re-enable the breakpoint by right-clicking on it and selecting Enable.
+
+The limits can be set based on:
+- Time (e.g., 1 Hour, 24 hours, a week)
+- Hit count (the number of times the breakpoint gets triggered)
```
Comment (ElDuderinos) [inline docs/breakpoints.md:30]: You wrote about the hit count in breakpoint-limits.md
Either choose one place or properly reference from here to there.

**Review by ElDuderinos — CHANGES_REQUESTED** [review]

**Review by ElDuderinos — APPROVED** [review]

**Pull request merged**